packages/realm-app-importer/src/AppImporter.ts

```diff
@@ -182,22 +191,50 @@ export class AppImporter {
     cp.execFileSync(cliPath, args, { stdio: "inherit" });
   }
 
+  private performLogIn() {
+    const { credentials } = this;
+    console.log(credentials);
+    if (credentials.kind === "api-key") {
+      return fetch(`${this.apiUrl}/auth/providers/mongodb-cloud/login`, {
+        method: "POST",
+        headers: { "content-type": "application/json" },
+        body: JSON.stringify({
+          username: credentials.publicKey,
```

Comment:
Is `username` the correct field name?  I don't know the API, just wanted to make sure this wasn't a typo, as this name doesn't really reflect what it is.

Reply:
Yeah - it's kind of weird that it's sending the public key like that, but I did check the `realm-cli` package: https://github.com/10gen/realm-cli/blob/master/internal/cloud/realm/auth.go#L22-L25